docs/web-ui.md

```diff
@@ -107,6 +107,51 @@ sizes and using executors for all partitions in an RDD or DataFrame.
 The Environment tab displays the values for the different environment and configuration variables,
 including JVM, Spark, and system properties.
 
+<p style="text-align: center;">
+  <img src="img/webui-env-tab.png"
+       title="Env tab"
+       alt="Env tab"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+This environment page has five parts, it is a useful place to check whether your properties have
+been set correctly.
+The first part 'Runtime Information' simply contains the [runtime properties](configuration.html#runtime-environment)
+like versions of Java and Scala.
+The second part 'Spark Properties' lists the [application properties](configuration.html#application-properties) like
+'spark.app.name' and 'spark.driver.memory'.
```

Comment:
You could back-tick-quote these, but it's not a big deal.

Reply:
Do you mean adding links for 'spark.app.name' and 'spark.driver.memory'?
It seems that there isn't an anchor link for a specific property in `https://spark.apache.org/docs/latest/configuration.html#application-properties`.

Reply:
No I just mean like `spark.app.name`